tgtg/__init__.py

```diff
@@ -19,22 +22,23 @@
     "TGTG/21.9.3 Dalvik/2.1.0 (Linux; Android 6.0.1; SM-G920V Build/MMB29K)",
 ]
 DEFAULT_ACCESS_TOKEN_LIFETIME = 3600 * 4  # 4 hours
+MAX_POLLING_TRIES = 24  # 24 * 5 seconds = 2 minutes
```

Comment:
We need another constant to define delay (5s) between retries

Reply:
Added!